packages/ts-transformers/package.json

```diff
@@ -57,7 +57,7 @@
   },
   "dependencies": {
     "ts-clone-node": "^1.0.0",
-    "typescript": "~4.7.4"
+    "typescript": "~5.0.0"
```

Comment:
Is this a breaking change? Thinking about lit.dev here.

Reply:
Yeah, marked as such